docs/releases/2.2.12.txt

```diff
@@ -9,4 +9,5 @@ Django 2.2.12 fixes several bugs in 2.2.11.
 Bugfixes
 ========
 
-* ...
+* Added the ability to handle ``.po`` files containing different plural
```

Comment:
Maybe: `Django 2.2.12 fixes a bug in 2.2.`, I don't think that we will decide to backport more bugfixes.

Reply:
OK, updated. 👍